### PR TITLE
Tidy the layout

### PR DESCRIPTION
**Pull request by philwareham**
Plugin requires Textpattern 4.6+, although I've only tested in 4.7 so far so I may need to PR some 4.6 fixes after first PR.

I'd like to get Composer install tested before I fix any 4.6 things, so please merge and I'll test that at the same time that I test 4.6 installs.

### CODE REVIEW

File: smd_thumbnail.php [hunk 1]
```diff
@@ -1454,21 +1453,31 @@ function smd_thumb_switch_active_done(data) {
...
-                    'sort'           => $sort,
-                    'dir'            => $dir,
-                    'crit'           => $crit,
-                    'search_method'  => $search_method,
-                ), array(
-                    'class'       => 'dlink destroy ui-icon ui-icon-close',
-                    'title'       => gTxt('delete'),
-                    'data-verify' => gTxt('smd_thumb_delete_confirm', array("{name}" => $row['name'])),
-                ));
+                $btnDel = tag(
```
Comment (Bloke) [inline smd_thumbnail.php]: Why an anchor inside a button? I can't get the delete to work now. Delete button just does... nothing.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "bloke/smd_thumbnail",
+  "description": "Multiple image thumbnails of arbitrary dimensions within Textpattern CMS",
+  "keywords": ["textpattern", "plugin", "images", "src-set"],
+  "homepage": "https://github.com/bloke/ied_plugin_composer",
```
Comment (Bloke) [inline composer.json]: /smd_thumbnail :-)

File: smd_thumbnail.php [hunk 3]
```diff
@@ -1270,96 +1268,71 @@ function smd_thumb_subsel(obj) {
...
             $out[] = $btnPnl;
             $out[] = '<div id="smd_thumb_batch"><span id="smd_thumb_bcurr"></span><span id="smd_thumb_btot"></span></div>';
             $out[] = '<form method="post" name="smd_thumb_multi_edit" id="smd_thumb_multi_edit" action="'.join_qs($qs).'">';
 
-            $out[] = inputLabel(
-                'smd_thumb_group_type',
-                $grpOpts.$btnGrp,
-                'smd_thumb_batch_preamble', '', array('class' => 'txp-form-field')
-            );
+            $out[] = '<p><label for="smd_thumb_group_type">'.gTxt('smd_thumb_batch_preamble').'</table>'.$grpOpts.$btnGrp.'</p>';
```
Comment (Bloke) [inline smd_thumbnail.php]: `</label>` not `</table>` Or have I missed a start table tag somewhere?

File: composer.json (hunk 2 above)
Comment (philwareham) [inline composer.json]: Fixed, sorry!

File: smd_thumbnail.php (hunk 1 above)
Comment (philwareham) [inline smd_thumbnail.php]: OK, it worked for me in Safari.

Anyway, I think the issue is with `function dLink()` itself - seems to have a couple of different ways of rendering a 'destroy' button that I wasn't aware about. I need to decipher that code and work out a way to make sure they are styled the same in my themes. Leave it with me for a few hours.

**Comment by philwareham** [comment]
Righto, I've reverted the destroy button and will fix that in my themes upon checking the code within core. So this is probably ready to merge unless you have other concerns.

**Comment by philwareham** [comment]
Great - if you can do the Packagist stuff and tag a beta, I can test. I have installed tom_image_grid and will create a separate PR for that (along with any Composer and 4.6 fixes) after first beta.

**Pull request merged**